queries.sql

```diff
@@ -35,4 +35,129 @@ SELECT *
 SELECT *
 	FROM "VET1".animals
 	WHERE
- 	weight_kg >=10.4 AND weight_kg <=17.3;
\ No newline at end of file
+ 	weight_kg >=10.4 AND weight_kg <=17.3;
+
+
+//PART2
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'unspecified';
+
+SELECT * FROM "VET1".animals
```

Comment:
- [ ] Great job on your queries to update the species column in the animal's table! 🎉 However, I'd like to point out a small oversight that can help enhance your code. 

   - From lines `50` to `61`, I noticed that you've written a query to update the species column to `unspecified` within a transaction, as required by the project requirements. However, I also noticed some incomplete code in this section that appears to be attempting to do the same thing. To avoid any confusion, I recommend removing this incomplete code from your implementation. 

Overall, your code looks solid, and this minor adjustment will make it even clearer and more maintainable. 
Happy coding! 😊🚀🍀